### PR TITLE
Add ETCD env vars for downgrade

### PR DESCRIPTION
**Pull request by krousey**
This is a partial fix for https://github.com/kubernetes/kubernetes/issues/43470

### CODE REVIEW

**Comment by ethernetdan** [comment]
@ixdy @skriss @mml could you PTAL at this release blocker?

File: test/e2e/framework/nodes_util.go [hunk 1]
```diff
@@ -62,8 +62,17 @@ func etcdUpgradeGCE(target_storage, target_version string) error {
 }
 
 func masterUpgradeGCE(rawV string) error {
+	env := os.Environ()
+	// TODO: Remove these variables when they're no longer needed for downgrades.
```
Comment (ixdy) [inline test/e2e/framework/nodes_util.go]: is there an associated issue tracking this?

**Comment by mml** [comment]
Way to reuse code!

/lgtm

File: test/e2e/framework/nodes_util.go (hunk 1 above)
Comment (krousey) [inline test/e2e/framework/nodes_util.go]: https://github.com/kubernetes/kubernetes/issues/43486 There is now.

**Pull request merged**